src/ipfs.ts

```diff
@@ -15,86 +15,87 @@ const config: {
   fetchTimeoutInSeconds: number;
 } = require('../json/ipfs-config.json');
 
-const requestHandler = new SidetreeIpfsService(config.fetchTimeoutInSeconds);
-const app = new Koa();
+void SidetreeIpfsService.create(config.fetchTimeoutInSeconds).then(requestHandler => {
```

Comment:
Not a fan of added layer of nesting for the remainder of the code. At least move `setKoaResponse()` out of the nesting.